src/Service/DataFetcher.php

```diff
@@ -73,4 +56,41 @@ public function getCurrencyApiByDateRange(string $dateFrom, string $dateTo)
             new \DateTimeImmutable($dateTo)
         );
     }
+
+    private function addTodayTransaction(string $currency): void
+    {
+        $beginningToday = (new \DateTimeImmutable())
+            ->setTime(0, 0, 0)
+            ->sub(new \DateInterval('P0Y0M0DT1H0M0S'));
```

Comment:
```suggestion
        $startOfToday = (new \DateTimeImmutable("00:00:00"))
            ->sub(new \DateInterval('PT1H'));
```

"P0Y0M0DT1H0M0S" багато зайвого, формат такий 
P - обовзяково
потім кількість років, місяців, днів
T - обовзяково якщо час вказуєш
кількість годин, хвилин секунд

виходит нам достатньо PT1H